src/__test__/generate-test-data.ts

```diff
@@ -21,7 +21,8 @@ export function generateLayout(
   nMeas: number,
   nRows: number,
   qColumnOrder: number[] = [],
-  qGrandTotalRow: Record<string, string>[] = []
+  qGrandTotalRow: Record<string, string>[] = [],
+  qEffectiveInterColumnSortOrder: number[] = [0]
```

Comment:
the default here is incorrect if you don't have one column, right? I think if we should have a fallback, then it should be correct

Reply:
if we dont have one column, then then table should not be rendered, right? 
So I think having one column means that table is sorted based on that one column and this default is fine? or am i  wrong in this case?

Reply:
no the default value is `Array.from(Array(nDims + nMeas).keys())`